app/icon.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
For now, calling it `close-icon` sounds like a good idea to me, since that's what it is. We can revisit it in the future.